package.json

```diff
@@ -43,7 +45,8 @@
     "toastr"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.0.0"
+    "ember-cli-babel": "^5.0.0",
+    "toastr": "^2.1.2"
```

Comment:
If this gets added to the user's project, it shouldn't be here (only in dev deps for dummy app).